### PR TITLE
Fixes #64365 and #35096 by getting rid of unused admin tooling.

### LINKED ISSUE #35096: Notes for ckey randomization don't get placed anymore
According to MSO this has been broken since 22/01/2018

22:45 <%MrStonedOne> stopped 2018-01-22 08:17:06

### LINKED ISSUE #64365: Notes for ckey randomization don't get placed anymore #35096
Comments
@optimumtact
 
Member
optimumtact commented on Jan 29, 2018
According to MSO this has been broken since 22/01/2018

22:45 <%MrStonedOne> stopped 2018-01-22 08:17:06

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Fixes #64365
Fixes #35096
## Why It's Good For The Game

closing issue reports good
this shit is unused for like 4 years

## Changelog
:cl:
fix: Fixes a few issues that resulted from unused administration tools.
/:cl:

### CODE REVIEW

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -576,7 +576,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if (src.holder && src.holder.rank)
 		admin_rank = src.holder.rank.name
 	else
-		if (!GLOB.deadmins[ckey] && check_randomizer(connectiontopic))
+		if (!GLOB.deadmins[ckey])
```
Comment (MrStonedOne) [inline code/modules/client/client_procs.dm:579]: This change doesn't do what you think it does

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
This pr as is breaks client connection db processing, including panic bunker.

please see line 579 and pm me on discord if you can't figure it out.

Also the config in config.txt should be removed

File: code/modules/client/client_procs.dm (hunk 1 above)
Comment (Timberpoes) [reply code/modules/client/client_procs.dm:579]: To help you a step forward goof:

Old: If they are not in the deadmin list AND they appear to be using a randomizer, we early return.
This condition only passes if they're using a CID randomizer and are not in the the deadmin list.

New: If they are not in the deadmin list, we early return.
This conidition will always pass if the player is not in the deadmin list.

The logic of your new code is totally wrong. It's gone from almost never being true, to almost always being true.

**Review by Timberpoes — CHANGES_REQUESTED** [review]

File: code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -576,7 +576,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
...
 			return
```
Comment (MrStonedOne) [inline code/modules/client/client_procs.dm:580]:
```suggestion

```

File: code/modules/client/client_procs.dm [hunk 3]
```diff
@@ -733,82 +733,6 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
...
-
-		//we sleep after telling the client to reconnect, so if we still exist something is up
-		log_access("Forced disconnect: [key] [computer_id] [address] - CID randomizer check")
-
-		qdel(src)
-		return TRUE
-
 /client/proc/cid_check_reconnect()
 	var/token = md5("[rand(0,9999)][world.time][rand(0,9999)][ckey][rand(0,9999)][address][rand(0,9999)][computer_id][rand(0,9999)]")
 	. = token
```
Comment (MrStonedOne) [inline code/modules/client/client_procs.dm:738]: you need to kill this proc and the one below it too. (but not add_system_note)

**Comment by MrStonedOne** [comment]
For somebody always bitching about how configs can't ever be removed because I won't allow it, you sure dropped the ball on your one chance to remove a config.

=P

**Comment by Iamgoofball** [comment]
sorry, just been busy with work shit recently

**Pull request closed without merging**